orcid-core/src/test/java/org/orcid/core/adapter/JpaJaxbGivenPermissionToAdapterTest.java

```diff
@@ -103,9 +106,11 @@ public void fromGivenPermissionToEntityToDelegationDetailsTest() {
         assertEquals(1, details.getDelegateSummary().getLastModifiedDate().getValue().getMonth());
         assertEquals(2016, details.getDelegateSummary().getLastModifiedDate().getValue().getYear());                
         assertNotNull(details.getDelegateSummary().getOrcidIdentifier());
+        assertTrue(details.getDelegateSummary().getOrcidIdentifier().getHost().equals("localhost") || details.getDelegateSummary().getOrcidIdentifier().getHost().equals("testserver.orcid.org"));
+        assertTrue(details.getDelegateSummary().getOrcidIdentifier().getUri().equals("http://localhost:8080/orcid-web/9999-9999-9999-9999") || details.getDelegateSummary().getOrcidIdentifier().getUri().equals("http://testserver.orcid.org/9999-9999-9999-9999"));
         assertEquals("9999-9999-9999-9999", details.getDelegateSummary().getOrcidIdentifier().getPath());
-        assertEquals("testserver.orcid.org", details.getDelegateSummary().getOrcidIdentifier().getHost());
-        assertEquals("http://testserver.orcid.org/9999-9999-9999-9999", details.getDelegateSummary().getOrcidIdentifier().getUri());
+        //assertEquals("testserver.orcid.org", details.getDelegateSummary().getOrcidIdentifier().getHost());
+        //assertEquals("http://testserver.orcid.org/9999-9999-9999-9999", details.getDelegateSummary().getOrcidIdentifier().getUri());
```

Comment:
Hey @TomDemeranville why 112 and 113 are commented? it looks like you are testing the same on 109 and 110 right? if so, lets remove 112 and 113 to avoid confusion, ok?